examples/guide/references.rs

```diff
@@ -0,0 +1,102 @@
+#![cfg_attr(verus_keep_ghost, verifier::exec_allows_no_decreases_clause)]
+
+#[allow(unused_imports)]
+use builtin::*;
+#[allow(unused_imports)]
+use builtin_macros::*;
+use vstd::prelude::*;
+
+verus! {
+
+
+fn immut()
+{
+// ANCHOR: immut
+let x: u32 = 0;
+let immut_ref_x = &x;
+assert(x == 0);
+assert(*immut_ref_x == 0);
+// ANCHOR_END: immut
+}
+
+// ANCHOR: mut
```

Comment:
Stylistically, we generally sort functions such that callees precede callers, so in this case, that would mean putting `modify_y` before `mutable_example`.